example.js

```diff
@@ -0,0 +1,43 @@
+// First let's import the logging module.
+var logging = require('./lib/log');
+
+// Now, let's create a module-level object.  The name
+// parameter is mandatory and should be the name of your
+// app.  The object passed in here is just a bunyan log
+// config object.
+var log = logging("my-example", {module: "main"});
```

Comment:
So do I want to do this in all files for my application?

What is file and app specific here of my-example and module: main?